RPIPs/RPIP-49.md

```diff
@@ -1,68 +1,127 @@
+
+The tokenomics rework will likely be split into two protocol upgrades, Saturn I and Saturn II. This RPIP describes the components that are expected to be included within each.
+
+This Informational RPIP and the tokenomics rework represent the best efforts of the Rocket Pool community contributors involved. Information within this RPIP should not be considered official word from the Rocket Pool core development team. 
+
+## Contents
+
+### [RPIP-43: Megapools](RPIP-43.md)
+
+A Megapool is a single contract that can be used as an ethereum withdrawal address for multiple validators.
+* This allows for much more gas-efficient usage of the Rocket Pool protocol for Node Operators.
+* This allows for the application of node-level penalities. 
+* Importantly, this Megapool specification allows Node Operators to create ETH-only validators. 
+
+Additionally, Megapools are required to facilitate the bond curve changes described in RPIP-42.
+
+### [RPIP-42: Bond curves](RPIP-42.md)
+
+The changes to bond curves allow Node Operators to provide smaller ETH bonds in a manner which does not impact the security of the Rocket Pool protocol. 
+* This gives Node Operators the option of dramatically increasing their capital efficiency.
+* This allows the Rocket Pool Protocol to support a greater amount of rETH. 
+
+In order for bonds to be lowered securely for the protocol, initial bonds for a node must be larger than the minimum bond. 
+
+### [RPIP-46: Universal Adjustable Revenue Split](RPIP-46.md)
+
+This change allows the ETH revenue income from borrowed ETH to be split between various targets. The four targets are:
+1. The Node Operator borrowing the ETH.
+2. Node Operators generally, proportional to their share of voting RPL.
+3. The surplus share, used to capture value to the RPL token.
+4. The rETH share, going to rETH holders. 
+
+The split is both adjustable by the pDAO, and may adjust automatically according to various heuristics.
+
+### RPL Value Capture
+
+Some form of value capture method will be included in the tokenomics rework package. Three options are being actively debated:
+* [RPL Burn](RPIP-45.md) - Use the surplus share to buy and burn RPL.
+* [RPL Buy & LP](RPIP-50.md) - Use the surplus share to buy RPL and deposit it in a liquidity pool. 
+* Direct the surplus share to Node Operators, proportional to their share of voting RPL.
+
+### [RPIP-44: Forced exits](RPIP-44.md)
+
+This change allows the Rocket Pool protocol to force-exit Node Operators under certain circumstances. It relies on [EIP-7002](https://eips.ethereum.org/EIPS/eip-7002) being adopted by the Ethereum Protocol. 
+* This allows Node Operators to easily exit their validators.
+* This allows the Rocket Pool Protocol to exit badly performing or malicious validators.
```

Comment:
badly performing is not currently covered

Reply:
Oops. Fixed.